horizontal-arm/arm-read-write/Program.cs

```diff
@@ -143,7 +137,7 @@ private static async Task SendDeviceToCloudMessagesAsync(CancellationToken token
                 await s_deviceClient.SendEventAsync(message);
 
                 // Print out the message.
-                Console.WriteLine("{0} > Sent message: {1}", DateTime.Now, telemetryDataString);
+                Console.WriteLine("{0} > Sent message: {1}", DateTime.UtcNow, telemetryDataString);
```

Comment:
While I'd agree with using UTC for service applications, for local apps having the time local to the person is better. Would you agree?

Reply:
Oh, of course. I'll change.